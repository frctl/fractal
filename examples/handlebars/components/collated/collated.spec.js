const fractal = require('../../fractal.config.js');

describe('tree', () => {
    beforeEach(async () => {
        await fractal.load();
    });

    it('renders collated components collated', async () => {
        const render = await fractal.components.find('@collated').render(null, null, { collate: true });
        expect(render).toMatchSnapshot();
    });
});
