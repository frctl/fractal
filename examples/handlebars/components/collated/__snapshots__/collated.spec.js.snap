// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collated renders collated components collated 1`] = `
"<!-- Start: @collated--default -->
<div>
    Collated component default
</div>
<!-- End: @collated--default -->

<!-- Start: @collated--one -->
<div>
    Collated component one
</div>
<!-- End: @collated--one -->

<!-- Start: @collated--two -->
<div>
    Collated component two
</div>
<!-- End: @collated--two -->

<!-- Start: @collated--three -->
<div>
    Collated component three
</div>
<!-- End: @collated--three -->
"
`;
